[FLINK-18186][doc] Various updates on standalone kubernetes document (#12690)
apache/flink: docs/ops/deployment/kubernetes.md

        {% highlight bash %}./bin/flink run -m <public-node-ip>:<node-port> ./examples/streaming/WordCount.jar{% endhighlight %}

> * Create a `NodePort` service on the query state service of taskmanager:

Reviewer: ```suggestion
You can also access the queryable state of TaskManager if you create a `NodePort` service for it:
```

Reviewer: by using `* ` you continue the list related to:
`You can then access the Flink UI via different ways:`

Author: Actually i want to continue the list. I think your suggestion also make sense. I will update it.

    1. Run `kubectl create -f taskmanager-query-state-service.yaml` to create the `NodePort` service on taskmanager. The example of `taskmanager-query-state-service.yaml` can be found in [appendix](#common-cluster-resource-definitions).

>     2. Run `kubectl get svc flink-taskmanager-query-state` to know the `node-port` of this service. Then `QueryableStateClient(<public-node-ip>, <node-port>)` could be used to submit the user queries.

Reviewer: ```suggestion
    2. Run `kubectl get svc flink-taskmanager-query-state` to know the `node-port` of this service. Then you can create the [QueryableStateClient(<public-node-ip>, <node-port>)]({{ site.baseurl }}/dev/stream/state/queryable_state.html#querying-state) to submit the state queries.
```

    kubectl delete -f flink-configuration-configmap.yaml
    # if created then also the rest service
    kubectl delete -f jobmanager-rest-service.yaml

>     kubectl delete -f taskmanager-query-state-service.yaml

Reviewer: ```suggestion
    # if created then also the queryable state service
    kubectl delete -f taskmanager-query-state-service.yaml
```

    component: jobmanager
{% endhighlight %}

> `taskmanager-query-state-service.yaml`. Optional service, that exposes the taskmanager `query-state` port as public Kubernetes node's port.

Reviewer: should we mention this service in `## Deploy Flink cluster on Kubernetes` chapter?
like we describe `jobmanager-rest-service.yaml`:
- after description of access REST API, we could mention that the `query-state` port can be accessed in a similar way.
- add the optional `kubectl create/delete -f taskmanager-query-state-service.yaml` command

> `taskmanager-query-state-service.yaml`. Optional service, that exposes the taskmanager `query-state` port as public Kubernetes node's port.

Reviewer: ```suggestion
`taskmanager-query-state-service.yaml`. Optional service, that exposes the taskmanager port to access the queryable state as a public Kubernetes node's port.
```

        ports:
        - containerPort: 6122
          name: rpc

>         - containerPort: 6125

Reviewer: Btw the user actually also needs a `Service` for the `query-state` port or something else, like we documented for REST. Maybe, this could be a follow-up improvement if not in this PR.

Author: I will add one more commit for your comments since it is a new added service.

        - name: jobmanager
          image: flink:{% if site.is_stable %}{{site.version}}-scala{{site.scala_version_suffix}}{% else %}latest{% endif %}
          env:

>           args: ["standalone-job", "--job-classname", "com.job.ClassName", "--job-id", "<job id>", "--fromSavepoint", "/path/to/savepoint", "--allowNonRestoredState", <job arguments>]

Reviewer: ```suggestion
          args: ["standalone-job", "--job-classname", "com.job.ClassName", <optional arguments>, <job arguments>] # optional arguments: ["--job-id", "<job id>", "--fromSavepoint", "/path/to/savepoint", "--allowNonRestoredState"]
```
